redux/cardSlice.jsx

```diff
@@ -0,0 +1,35 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+
+export const fetchUsers = createAsyncThunk('fetchUsers', async () => {
+    const response = await axios.get("https://jsonplaceholder.typicode.com/users");
+    return response.data;
+});
+
+const initialState = {
+  users: [],
+  loading: false,
+  error: null,
+};
+
+export const cardSlice = createSlice({
+  name: "card",
+  initialState,
+  extraReducers: (builder) => {
```

Comment:
what is the use of extra reducers here ??

Reply:
extrareducers is like reducers as we are using thunk for asynchronous task here so to handle the cases we are using extra reducers here